smac/runhistory/runhistory.py

```diff
@@ -49,9 +49,16 @@ class RunHistory(object):
     ---------
     aggregate_func: callable
         function to aggregate perf across instances
+    overwrite_existing: bool
+        allows to overwrites old results if pairs of 
+        algorithm-instance-seed were measured  
+        multiple times
     '''
 
-    def __init__(self, aggregate_func):
+    def __init__(self, 
+                 aggregate_func: typing.Callable,
+                 overwrite_existings: bool=False
```

Comment:
This should be "overwrite_existing" without the 's' as in the docstring. "overwrite_existing_runs" would be even better